app/models/review.rb

```diff
@@ -11,4 +11,22 @@ class Review < ActiveRecord::Base
 
   validates :rating, presence: true,
              inclusion: { in: RATINGS }
+
+  def total_votes
+    votes.sum(:score)
+  end
+
+  def increment_vote(user)
+    fetch_vote(user).increment
+  end
```

Comment:
Where are you preventing a user from repeatedly voting for the same review? If you're incrementing the score everytime then it is possible for the vote to be more than just 1 or -1.